doc/src/python.rst

```diff
@@ -70,80 +77,101 @@ Examples
        lmp.command("pair_style lj/cut ${cut}")   # LAMMPS commands
        lmp.command("pair_coeff * * 1.0 1.0")
        lmp.command("run 100")
-    """
+   """
+
+   python source funcdef.py
+
+   python source here "from lammps import lammps"
+
 
 Description
 """""""""""
 
-Define a Python function or execute a previously defined function or
-execute some arbitrary python code.
+The *python* command allows to interface LAMMPS with an embedded Python
```

Comment:
allows interfacing
(doesn't seem like all my previous comments appeared to you?)